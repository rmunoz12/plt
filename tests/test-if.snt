@setup {

}

@turns {

func void begin() {
    if (5 > 0) {
        print("if ok"); print("\n");
    }
    if (2 < 0) {
        print("if error"); print("\n");
    } else {
        print("if else ok"); print("\n");
    }
    if (2 < 0) {
        print("if error"); print("\n");
    } elif (3 > 0) {
        print("if elif ok"); print("\n");
    }

    end;
}

}
