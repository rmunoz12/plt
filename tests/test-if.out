if ok
if else ok
if elif ok
